packages/ui/src/core/image-comparison-slider/image-comparison-slider.props.ts

```diff
@@ -15,6 +15,7 @@ export interface ImageComparisonSliderProps extends React.HTMLAttributes<HTMLDiv
   leftImgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
 	rightImgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
 	fallbackPreviewProps?: FallbackPreviewProps,
+	footerProps?: { leftText?: string, rightText?: string, hideFooter?: boolean };
```

Comment:
I see that we only have tree properties here but we destructing restFooterProps.

Reply:
because if someone wants to pass any other props to the footer